.github/renovate.json

```diff
@@ -42,13 +42,171 @@
       "pinDigests": false
     },
     {
-      "description": "Do NOT generate PRs for major or minor go dependency updates ",
+      "description": "k8s go: minor and patch updates in main",
+      "enabled": true,
+      "groupName": "k8s-go main",
+      "matchDatasources": [
+        "go"
+      ],
+      "matchPackagePrefixes": [
+        "k8s.io/api",
```

Comment:
I thought there was a way we could use the group preset and override to include what we just need.  Let's can pare it down to just these three:
	k8s.io/api
	k8s.io/apimachinery
	k8s.io/client-go